CHANGELOG.md

```diff
@@ -1,2 +1,13 @@
-# Next release
-* Gecko: Return `Android` instead of `Android 8.0.0` for `user_agent.platform` when the version of the Android is given
+# 1.0.0
+
+Due to inactivity on upstream we decided to treat this as a real, independent fork. The following changes are added to this version
+
+* [#21](https://github.com/art19/useragent/pull/21) Improved support for watchOS
+  * Replace former implementation of Apple Watch browers with new AirTraffic.framework and Pod2Watch
+  * Add support for AirTraffic.framework on Apple Watch
+  * Add support for Pod2Watch on Apple Watch
+  * AppleCoreMedia now detects the Apple Watch operating system
+  * Converted this gem to work as a real fork and bumped the version to 1.0.0
+* [#20](https://github.com/art19/useragent/pull/20) Gecko android platform
+* [#19](https://github.com/art19/useragent/pull/19) Add support for 'Edge Anaheim'
+* [#18](https://github.com/art19/useragent/pull/18) Upgrade to Ruby 2.7 (PLAT-425)
```

Comment:
No need to use our Jira reference here because this gem is public and Jira is not